@here/harp-omv-datasource/lib/OmvRestClient.ts

```diff
@@ -301,6 +317,8 @@ export class OmvRestClient implements DataProvider {
             case APIFormat.MapboxV4:
             case APIFormat.XYZOMV:
             case APIFormat.XYZMVT:
+            case APIFormat.XYZSpace:
+               return AuthenticationTypeAccessToken;
```

Comment:
This additional return is not needed. The case will just fall through to line 323

Reply:
Done